plugins/hydra_submitit_launcher/tests/test_submitit_launcher.py

```diff
@@ -55,7 +55,35 @@ class TestSubmititLauncher(LauncherTestSuite):
                 },
             },
             ["-m"],
-        )
+        ),
+        # auto queue
+        (
+            {
+                "defaults": [
+                    {"hydra/launcher": "submitit"},
```

Comment:
Seems to be worth adding a submit_local config option that would do the right thing.

Reply:
Not sure what you mean here

Reply:
The current design of this plugin is that one config can do all three modes by selecting the mode with queue_type.

An alternative design is that we have 3 different config group options:
```
hydra/launcher:
  submitit_slurm
  submitit_local
  submitit_auto
```

and have the user select the one they need. it should come with the basic defaults  for that mode.
This is what we ended up with for the Ray launcher (WIP in PR) from @jieru-hu.
This can be a followup diff, it is a user facing change and we should think about it carefully.

Reply:
I wouldn't be the best person for this, it's up to you but probably better not mix it with this diff